system_test/common/aest_sync_SUITE.erl

```diff
@@ -210,23 +210,23 @@ new_node_joins_network(Cfg) ->
     %% Starts a chain with two nodes
     start_node(old_node1, Cfg),
     start_node(old_node2, Cfg),
-    T0 = erlang:system_time(seconds),
```

Comment:
Ouch...

Ah, wait, no, `seconds` is just deprecated, not wrong! Then this is just a good update.